hw/dv/tools/testplans/shadow_reg_errors_testplan.hjson

```diff
@@ -0,0 +1,30 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  entries: [
+    {
+      name: shadow_reg_errors
+      desc: '''
+            Verify shadow registers' update and storage errors.
+            - Randomly choose to issue reset. A reset will clear all the internal stored values and
```

Comment:
very nit: "choose" is IMO a user option - not a randomization.
maybe instead we could say:

> Resets are Issued at random to clear all the internal stored values and  phases trackers in shadow registers